product_docs/docs/pgd/5/sequences.mdx

```diff
@@ -229,6 +230,9 @@ support renaming `galloc` sequences or moving them to another namespace or
 renaming the namespace that contains a `galloc` sequence. Be
 mindful of this limitation while designing application schema.
 
+Additionally, after restoring a node from a backup, all galloc sequences must manually 
```

Comment:
Not sure this is correct.

Additionally, after restoring a node from a backup, all galloc sequences must manually have their next value set to the maximum value of the related column. See ["Set a new start value for the sequence"](#2-set-a-new-start-value-for-the-sequence).

Reply:
@wind39 This is what you told me and what we did for the customer. please confirm

Reply:
@djw-m is correct. We didn't set the maximum value of the sequence to the maximum value of the column.

@matthew123987 , check your text: `have their maximum value set to the maximum value of`

It's a paste-o, well spotted by @djw-m .

Reply:
@wind39 have popped in my suggested text from above. Would be good to get a review and approval on this so we can merge (and push the changes into 5.6 content too, as that's the new latest branch).